go.mod

```diff
@@ -51,13 +51,13 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/ini.v1 v1.42.0 // indirect
 	gotest.tools v2.2.0+incompatible // indirect
-	k8s.io/api v0.0.0-20181005203742-357ec6384fa7
-	k8s.io/apimachinery v0.0.0-20180913025736-6dd46049f395
+	k8s.io/api v0.0.0-20191010143144-fbf594f18f80
+	k8s.io/apimachinery v0.0.0-20191016060620-86f2f1b9c076
 	k8s.io/apiserver v0.0.0-20181005205051-9f398e330d7f
 	k8s.io/client-go v0.0.0-20181005204318-cb4883f3dea0
+	k8s.io/cluster-bootstrap v0.0.0-20191016114958-5aa5f2bc1590
```

Comment:
This is k8s.io/cluster-bootstrap@kubernetes-1.17, please pin it to the version that is used for all other k8s dependencies in this project - kubernetes-1.12.1

Reply:
That was a nice catch @ialidzhikov . Thanks for that.

Reply:
There is no kubernetes-1.12.1 so I switch it to kubernetes-1.13.1